homeassistant/components/fan/xiaomi_miio.py

```diff
@@ -196,50 +437,115 @@ def device_state_attributes(self):
 
     @property
     def is_on(self):
-        """Return true if fan is on."""
+        """Return true if device is on."""
         return self._state
 
-    @asyncio.coroutine
-    def _try_command(self, mask_error, func, *args, **kwargs):
-        """Call an air purifier command handling error messages."""
+    @staticmethod
+    def _extract_value_from_attribute(state, attribute):
+        from enum import Enum
```

Comment:
Normal imports (not related to backend libraries) should be done on top of the file.